src/test/java/Qa2HomeworkOne.java

```diff
@@ -7,33 +7,50 @@
 import org.openqa.selenium.WebElement;
 import java.util.List;
 
+import static org.junit.jupiter.api.Assertions.*;
+
+
 public class Qa2HomeworkOne {
 
     private final By ACCEPT_COOKIES_BTN = By.xpath(".//button[@mode = 'primary']");
-    private final By HEADLINES_LOCATOR_CLASS = By.xpath(".//span[@class = 'list-article__headline']");
-    private final By COMMENTS_LOCATOR_IMAGE_BTN = By.xpath(".//img[@src = '/v5/img/icons/comment-v2.svg']");
-    private final By COMMENTS_LOCATOR_CLASS = By.xpath(".//span[@class = 'list-article__comment section-font-color']");
-    private final By LOGO_LOCATOR_IMAGE = By.xpath(".//img[@src = 'https://f.pmo.ee/logos/4133/7b1236dab95abca45083322781760e97.svg']");
-    private final By MENUTOP_LOCATOR_CLASS = By.xpath(".//div[@class = 'menu-items menu-items--top']");
-    private final By ITEMMENUTOP_LOCATOR_CLASS = By.xpath(".//a[@class = 'menu-item']");
-
-    // Тест 1: Задание: найти первую статью и перейти в комментарии.
-    @Test
-    public void homeworkOneTaskOne() {
+    private final By HEADLINE_CLASS = By.className("list-article__headline");
+    private final By COMMENT_CLASS = By.className("list-article__comment");
+    private final By COMMENT_BTN = By.xpath(".//img[@src = '/v5/img/icons/comment-v2.svg']");
+    private final By LOGO = By.xpath(".//img[@src = 'https://f.pmo.ee/logos/4133/7b1236dab95abca45083322781760e97.svg']");
+    private final By TOP_MENU_CLASS = By.className("menu-items--top");
 
-        System.setProperty("webdriver.chrome.driver", "c://chromedriver.exe");
-        WebDriver browserWindow = new ChromeDriver();
-        browserWindow.manage().window().maximize();
-        browserWindow.get("http://tvnet.lv");
-        WebDriverWait wait = new WebDriverWait(browserWindow, 10);
+    private WebDriver driver = null;
```

Comment:
how does it work?

Reply:
`@BeforeEach` assigns a new driver object before each test, so it is only `null` at the beginning - when the test class object is created.